app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -68,24 +69,24 @@ class RegisterController @Inject() (val messagesApi:           MessagesApi,
 
           result.fold[Result](
             { e ⇒
-              logger.warn(s"For NINO [$nino]: Could not write confirmed email: $e")
+              logger.warn(s"For NINO [$nino]: Could not write confirmed email: $e", nino)
```

Comment:
This doesn't seem to follow your new logging 'pattern'

Reply:
Yup, I missed that one - will change it now!